api/grpc/types/api.pb.go

```diff
@@ -2407,169 +2415,170 @@ var _API_serviceDesc = grpc.ServiceDesc{
+	0x42, 0x45, 0xda, 0x5b, 0xd2, 0x9c, 0x1b, 0xa5, 0xed, 0x18, 0xbb, 0x44, 0x61, 0x28, 0x45, 0x7e,
+	0x0a, 0x3b, 0x84, 0x8e, 0xae, 0x02, 0xc2, 0xe7, 0xe5, 0xad, 0xbd, 0x6a, 0x9d, 0x2f, 0x02, 0xc2,
+	0xab, 0xc2, 0xd4, 0x3a, 0xa7, 0x98, 0x8d, 0x2b, 0xeb, 0x5c, 0x7f, 0xd5, 0x3a, 0x8f, 0x25, 0x47,
+	0x29, 0xe7, 0x29, 0xf4, 0x09, 0x9d, 0xb7, 0xa7, 0x79, 0xa3, 0x94, 0x4d, 0x42, 0xab, 0xb6, 0x1c,
+	0x42, 0x3f, 0xc3, 0x21, 0xa7, 0xcc, 0x8e, 0x85, 0x8d, 0x1b, 0x65, 0x6c, 0x69, 0x06, 0x23, 0xc4,
+	0xfb, 0x12, 0x3a, 0x3f, 0xce, 0xc7, 0x98, 0xc7, 0xe7, 0x26, 0xe7, 0xff, 0xdb, 0x65, 0xe6, 0x5f,
+	0x75, 0x68, 0x1f, 0x8e, 0x19, 0xcd, 0xd3, 0x4a, 0xd5, 0x56, 0x39, 0xbc, 0x50, 0xb5, 0x25, 0x8d,
+	0xac, 0xda, 0x8a, 0xfa, 0x43, 0xe8, 0xa8, 0x7b, 0x8d, 0x66, 0x50, 0x55, 0x08, 0x2d, 0x26, 0x7d,
+	0x71, 0x8f, 0x52, 0x6c, 0x07, 0xfa, 0x8e, 0xa8, 0xb9, 0xaa, 0xd5, 0xa8, 0x74, 0x93, 0x0f, 0xe7,
+	0x65, 0xd6, 0x1d, 0x41, 0x77, 0xa2, 0x7c, 0xa3, 0xb9, 0x54, 0x00, 0xbe, 0x53, 0x18, 0x57, 0xae,
+	0x61, 0xdf, 0xf6, 0xa1, 0x72, 0x75, 0x67, 0x62, 0xbb, 0xf5, 0x7d, 0x00, 0xd1, 0x34, 0x8f, 0x8a,
+	0x42, 0x65, 0xff, 0x22, 0x30, 0x27, 0x84, 0x6a, 0xac, 0xe5, 0x70, 0x70, 0x0a, 0xfd, 0x05, 0x99,
+	0x4b, 0xca, 0xd4, 0xb7, 0xec, 0x32, 0x55, 0x5e, 0x9c, 0x6c, 0x56, 0xbb, 0x76, 0xfd, 0xb5, 0xa6,
+	0x1e, 0x0d, 0xca, 0x57, 0xdc, 0xc7, 0xd0, 0x4d, 0x54, 0xf3, 0x65, 0x36, 0xc0, 0xbe, 0x81, 0xd9,
+	0x8d, 0x99, 0xdf, 0x49, 0xec, 0x36, 0xed, 0x43, 0xe8, 0x84, 0xd2, 0x03, 0x4b, 0x37, 0xc2, 0x72,
+	0x8e, 0xdf, 0x0e, 0xad, 0xdd, 0xae, 0x34, 0x8a, 0xce, 0xd7, 0x69, 0x14, 0xf5, 0xbb, 0xdf, 0xaa,
+	0x5f, 0x1a, 0x07, 0xff, 0x5c, 0x87, 0xc6, 0x93, 0xe1, 0x11, 0x3a, 0x83, 0xad, 0xf9, 0x3f, 0x82,
+	0xe8, 0xbe, 0x36, 0x6b, 0xc5, 0x5f, 0xc4, 0xc1, 0x83, 0x95, 0xf3, 0xba, 0x65, 0xbf, 0x85, 0x7c,
+	0xd8, 0x9c, 0xfb, 0xff, 0x83, 0x8a, 0xa3, 0x66, 0xf9, 0x3f, 0xb6, 0xc1, 0xfd, 0x55, 0xd3, 0xb6,
+	0xcc, 0xb9, 0x3b, 0x82, 0x91, 0xb9, 0xfc, 0xb5, 0xc5, 0xc8, 0x5c, 0x75, 0xb5, 0xb8, 0x85, 0xbe,
+	0x0f, 0xeb, 0xea, 0x8f, 0x10, 0x2a, 0x2e, 0x2e, 0x95, 0x7f, 0x4d, 0x83, 0x3b, 0x73, 0x58, 0xc3,
+	0xf8, 0x1c, 0xba, 0x95, 0xdf, 0x88, 0xe8, 0x8d, 0x8a, 0xae, 0xea, 0x0f, 0xa5, 0xc1, 0x9b, 0xcb,
+	0x27, 0x8d, 0xb4, 0x43, 0x80, 0xf2, 0xa7, 0x01, 0x72, 0x35, 0xf5, 0xc2, 0x8f, 0xa9, 0xc1, 0xbd,
+	0x25, 0x33, 0x46, 0xc8, 0x19, 0x6c, 0xcd, 0x3f, 0xe0, 0xa3, 0x39, 0xaf, 0xce, 0x3f, 0x9f, 0x9b,
+	0xad, 0x5c, 0xf9, 0xf2, 0x2f, 0xc5, 0xce, 0x3f, 0xcb, 0x1b, 0xb1, 0x2b, 0x7e, 0x0a, 0x18, 0xb1,
+	0x2b, 0xdf, 0xf3, 0x6f, 0xa1, 0xcf, 0xa1, 0x57, 0x7d, 0xe7, 0x46, 0x85, 0x93, 0x96, 0x3e, 0xf4,
+	0x0f, 0xde, 0x5a, 0x31, 0x6b, 0x04, 0x7e, 0x00, 0x6b, 0xea, 0x01, 0xbb, 0x48, 0x47, 0xfb, 0xdd,
+	0x7b, 0xb0, 0x5d, 0x45, 0x1a, 0xae, 0x87, 0xb0, 0xae, 0x6e, 0x97, 0x26, 0x00, 0x2a, 0x97, 0xcd,
+	0x41, 0xc7, 0xc6, 0x7a, 0xb7, 0x1e, 0xd6, 0x0a, 0x3d, 0x59, 0x45, 0x4f, 0xb6, 0x4c, 0x8f, 0xb5,
+	0x39, 0xe7, 0xeb, 0x32, 0x5d, 0x1f, 0xfd, 0x3b, 0x00, 0x00, 0xff, 0xff, 0xb8, 0x44, 0xf5, 0x97,
+	0xd0, 0x1f, 0x00, 0x00,
```

Comment:
@mlaventure any idea if these values are platform dependent?
First I generated them on macOS and now, when I run `make protoc` on ubuntu, I see these values change.